CRM/Upgrade/Incremental/php/FourSeven.php

```diff
@@ -1197,4 +1217,24 @@ protected function checkImageUploadDir() {
     return $config->imageUploadDir && $config->imageUploadURL && $check->isDirAccessible($config->imageUploadDir, $config->imageUploadURL);
   }
 
+  /**
+   * (Queue Task Callback)
+   *
+   * Execute a single SQL query.
+   *
+   * @param CRM_Queue_TaskContext $ctx
+   * @param string $sql
+   *   An SQL template. May include tokens `%1`, `%2`, etc.
+   * @param array $vars
+   *   List of SQL parameters, as used by executeQuery().
+   *
+   * @return bool
+   *
+   * @see CRM_Core_DAO::executeQuery
+   */
+  public static function task_executeQuery($ctx, $sql, $vars) {
```

Comment:
Maybe I missed it, but it doesn't look like you're using this anywhere. Are you?